pkg/services/search/sorting.go

```diff
@@ -29,6 +31,8 @@ type SortOption struct {
 	Name        string
 	DisplayName string
 	Description string
+	Index       int
+	MetaName    string
```

Comment:
One problem here, and I mean this in the "we'll probably see this as a request in a few months" not as "we should fix this before merging", is that we don't support plural and singular differentiation here.

The way I'd prefer to solve this is (_not in this PR_) to have a type specifically for this kind of value+text combinations which would make future internationalization and localization efforts easier 🙂

Naïvely I'm thinking something that could start out as

```go
type IntUnit struct {
    Unit string
    // String function uses Unit if SingularUnit is empty.
    SingularUnit string
    Value int64
}

type (u IntUnit) String() string {
    if u.Value == 1 && u.SingularUnit != "" {
        return fmt.Sprintf("%d %s", u.Value, u.SingularUnit)
    }
    return fmt.Sprintf("%d %s", u.Value, u.Unit)
}
``` 

This is implemented in a sufficient number of places that I guess it should take ten minutes or less to find a blog post outlining a much better solution. (even before accounting for internationalization)